libraries/botbuilder-dialogs-adaptive/package.json

```diff
@@ -28,6 +28,12 @@
     }
   },
   "dependencies": {
+    "@microsoft/recognizers-text-sequence": "~1.1.4",
```

Comment:
I don't know why this file is changed. Is this part of the ESLint work?

Reply:
What is the response for this?, we should add it to the PR's description

Reply:
ESLint suggests moving them from devDependencies to dependencies